src/main/java/pl/touk/sputnik/Test.java

```diff
@@ -5,8 +5,9 @@
     public boolean test = false;
 
     public int method() {
+      String str = null;
         if (test) {
-            return 1;
+            return str.length() + 1;
```

Comment:
\[FindBugs\] WARNING: NP: Null pointer dereference of str in pl\.touk\.sputnik\.Test\.method\(\)